packages/cms-cli/commands/logs.js

```diff
@@ -97,78 +116,113 @@ const tailLogs = async ({
-        COMMAND_NAME,
-        {
-          latest,
-          file,
-        },
-        portalId
-      );
-
-      logger.debug(
-        `Getting ${
-          latest ? 'latest ' : ''
-        }logs for function with path: ${functionPath}`
-      );
-
-      const functionResp = await getFunctionByPath(
+const action = async (args, options) => {
+  loadAndValidateOptions(options);
+
+  const { functionPath } = args;
+  const { latest, file, follow, compact } = options;
+  let logsResp;
+  const portalId = getPortalId(options);
+
+  trackCommandUsage(COMMAND_NAME, { latest, file }, portalId);
+
+  logger.debug(
+    `Getting ${
+      latest ? 'latest ' : ''
+    }logs for function with path: ${functionPath}`
+  );
+
+  const functionResp = await getFunctionByPath(portalId, functionPath).catch(
+    e => {
+      logApiErrorInstance(e, {
+        functionPath,
         portalId,
-        functionPath
-      ).catch(e => {
-        logApiErrorInstance(e, {
-          functionPath,
-          portalId,
-        });
-        process.exit(1);
       });
+      process.exit(1);
+    }
+  );
+
+  logger.debug(`Retrieving logs for functionId: ${functionResp.id}`);
 
-      logger.debug(`Retrieving logs for functionId: ${functionResp.id}`);
-
-      if (follow) {
-        await tailLogs({
-          functionId: functionResp.id,
-          functionPath,
-          portalId,
-          portalName: program.portal,
-          compact,
-        });
-      } else if (latest) {
-        logsResp = await getLatestFunctionLog(portalId, functionResp.id);
-      } else {
-        logsResp = await getFunctionLogs(portalId, functionResp.id);
-      }
-
-      if (logsResp) {
-        return outputLogs(logsResp, {
-          compact,
-        });
-      }
+  if (follow) {
+    await tailLogs({
+      functionId: functionResp.id,
+      functionPath,
+      portalId,
+
+      Name: options.portal,
+      compact,
     });
+  } else if (latest) {
+    logsResp = await getLatestFunctionLog(portalId, functionResp.id);
+  } else {
+    logsResp = await getFunctionLogs(portalId, functionResp.id);
+  }
+
+  if (logsResp) {
+    return outputLogs(logsResp, {
+      compact,
+    });
+  }
+};
 
-  addPortalOptions(program);
-  addLoggerOptions(program);
-  addHelpUsageTracking(program, COMMAND_NAME);
+// Yargs Configuration
+const command = `${COMMAND_NAME} <path>`;
+const describe = DESCRIPTION;
+const builder = yargs => {
+  yargs.positional('path', {
+    describe: 'Path to serverless function',
+    type: 'string',
+  });
+  yargs.option('latest', {
+    alias: 'l',
+    describe: 'retrieve most recent log only',
+    type: 'boolean',
+  });
+  yargs.option('compact', {
+    alias: 'c',
+    describe: 'output compact logs',
+    type: 'boolean',
+  });
+  yargs.option('tail', {
+    alias: ['t', 'follow', 'f'],
+    describe: 'tail logs',
+    type: 'boolean',
+  });
+
+  addConfigOptions(yargs, true);
+  addPortalOptions(yargs, true);
+  addLoggerOptions(yargs, true);
```

Comment:
Similar to [my other comment](https://github.com/HubSpot/hubspot-cms-tools/pull/282#discussion_r470200845), can we add `addHelpUsageTracking` here?

Reply:
I think `--help` and tracking is handled in the top-level `cli.js` setup. See https://github.com/HubSpot/hubspot-cms-tools/blob/master/packages/cms-cli/bin/cli.js#L42-L44